be/src/vec/functions/function_timestamp.cpp

```diff
@@ -702,7 +702,7 @@ struct UnixTimeStampStrImpl {
         std::tie(col_right, format_const) =
                 unpack_if_const(block.get_by_position(arguments[1]).column);
 
-        auto col_result = ColumnDecimal<Decimal64>::create(input_rows_count, 0);
+        auto col_result = ColumnDecimal<Decimal64>::create(input_rows_count, 6);
```

Comment:
why not using scale from planner? like `block.get_by_position(result).type->get_scale()`

Reply:
Since the return type is fixed,
static DataTypePtr get_return_type_impl(const ColumnsWithTypeAndName& arguments) {
    return make_nullable(std::make_shared<DataTypeDecimal<Decimal64>>(16, 6));
}

Reply:
> why not using scale from planner? like `block.get_by_position(result).type->get_scale()`

Since the return type is fixed,
static DataTypePtr get_return_type_impl(const ColumnsWithTypeAndName& arguments) {
    return make_nullable(std::make_shared<DataTypeDecimal<Decimal64>>(16, 6));
}

Reply:
does FE have same assumption?

Reply:
> does FE have same assumption?

    private static final List<FunctionSignature> SIGNATURES = ImmutableList.of(
            FunctionSignature.ret(IntegerType.INSTANCE).args(),
            FunctionSignature.ret(DecimalV3Type.WILDCARD).args(DateTimeV2Type.SYSTEM_DEFAULT),
            FunctionSignature.ret(IntegerType.INSTANCE).args(DateV2Type.INSTANCE),
            FunctionSignature.ret(IntegerType.INSTANCE).args(DateTimeType.INSTANCE),
            FunctionSignature.ret(IntegerType.INSTANCE).args(DateType.INSTANCE),
            FunctionSignature.ret(DecimalV3Type.createDecimalV3Type(16, 6)).args(VarcharType.SYSTEM_DEFAULT,
                    VarcharType.SYSTEM_DEFAULT),
            FunctionSignature.ret(DecimalV3Type.createDecimalV3Type(16, 6)).args(StringType.INSTANCE,
                    StringType.INSTANCE)
    );

Reply:
what if this signature changed?

Reply:
> what if this signature changed?

The range of UNIX_TIMESTAMP is fixed and cannot be changed.